report/IDE-config.txt

```diff
@@ -0,0 +1,59 @@
+MegaLinter can help you to define the same linter configuration locally
```

Comment:
Why is this entire directory still here?

Reply:
I think it was created by the old megalinter, did not add that manually. It is removed now